tests/Issues/IssueVarConstantNewline/Fixture/fixture.php.inc

```diff
@@ -23,30 +16,12 @@ namespace Rector\Core\Tests\Issues\IssueVarConstantNewline\Fixture;
 
 class Fixture
 {
-    /**
-     * @var string
-     */
-    public const A = 'a';
-
-    /**
-     * @var string
-     */
     public const B = 'b';
```

Comment:
@samsonasik I broke this test case. I tried to figure out how exactly it works and could not get my head around it.
Could I ask you to handle this one after this PR gets merged?

Reply:
I will try.

Reply:
https://github.com/rectorphp/rector-src/pull/2629